lib/find.coffee

```diff
@@ -135,10 +135,18 @@ module.exports =
   createViews: ->
     return if @findView?
 
-    history = {@findHistory, @replaceHistory, @pathsHistory}
+    findBuffer = new TextEditor(mini: true).getBuffer()
```

Comment:
I'm not fully understanding what's going on here... looks like you're creating these editors just to get access to their buffers? Why not just get access to the buffer of the editors you actually create?